src/python/pants/backend/python/goals/run_pex_binary_integration_test.py

```diff
@@ -281,3 +281,31 @@ def test_run_script_from_3rdparty_dist_issue_13747() -> None:
         result = run_pants(args)
         result.assert_success()
         assert SAY in result.stdout.strip()
+
+
+def test_pass_extra_pex_cli_subsystem_args() -> None:
```

Comment:
Another option would be adding a mocked test to `pex_cli_util.py`, rather than only testing the `pants run ...` version of this. In particular, this ends up being a very heavy-weight test, because it's having to do a full `run_pants` call (level 4 in https://www.pantsbuild.org/2.23/docs/writing-plugins/the-rules-api/testing-plugins).

What do you think?

Reply:
I've added a unit test to make sure that `Process` object created does have the `--pex-cli-args` flags passed which exercises the modified rule (where those arguments are added). I think that having just unit test is probably enough since this is what verifies that arguments are going to be passed. How pex is going to handle them is probably outside of the scope being tested, but I think having this integration (in addition to the most recently added unit test) would be beneficial, but I am not sure about that. Happy to go either way.